src/language/printer.js

```diff
@@ -84,7 +84,9 @@ const printDocASTReducer: any = {
   FloatValue: ({ value }) => value,
   StringValue: ({ value, block: isBlockString }, key) =>
     isBlockString
-      ? printBlockString(value, key === 'description' ? '' : '  ')
+      ? isPrintableBlockString(value)
+        ? printBlockString(value, key === 'description' ? '' : '  ')
+        : ''
```

Comment:
You don't need to change `print` if AST node marked as block string we should print it only as block string.
You need to change only `printSchema` please use test case from original issue.